package.json

```diff
@@ -107,7 +107,7 @@
 		"lint": "run-s -s lint:*",
 		"lint:config-defaults": "node bin/validate-config-keys.js",
 		"lint:css": "stylelint \"**/*.scss\" --syntax scss",
-		"lint:js": "yarn run install-if-deps-outdated && eslint --ext .js --ext .jsx --ext .ts --ext .tsx --cache .",
+		"lint:js": "yarn run install-if-deps-outdated && eslint --cache .",
```

Comment:
In case you are wondering:
*  `.js` will be matched by default
*  `.jsx` is declared in the new override
* `.ts` and `.tsx` are declared in https://github.com/Automattic/wp-calypso/blob/7c7d77fc22d7859235cc42d959b66762ade485fd/.eslintrc.js#L98